src/katgpucbf/fgpu/main.py

```diff
@@ -185,16 +185,17 @@ def field_callback(kws: WidebandOutputDict, key: str, data: str) -> None:
     try:
         kws: WidebandOutputDict = {}
         _parse_stream(value, kws, field_callback)
+        kws = {"taps": DEFAULT_TAPS, "w_cutoff": DEFAULT_W_CUTOFF, **kws}  # type: ignore
```

Comment:
Very interesting, the presence of `taps` or `w_cutoff` in `kws` overwrites (?) the default accommodation here. I did not know that was a thing! i.e. Specifying a key's value twice in dict instantiation and it takes the last mention.